### PR TITLE
Fix for cuda_compiler_version not found on local builds

### PR DESCRIPTION
**Pull request by mmcauliffe**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mmcauliffe** [comment]
Resolves #41

File: recipe/meta.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ source:
 build:
   number: 1
   # Bug in for cuda 11.1 with c++17 on linux: https://forums.developer.nvidia.com/t/nvc-20-9-fails-to-compile-code-instantiating-any-std-tuple-with-gcc-10-2-on-c-17/160987
-  skip: true  # [cuda_compiler_version == "10.2" or (linux and cuda_compiler_version == "11.1")]
+  skip: true  # [cuda_compiler_version == "12.0" or cuda_compiler_version == "10.2" or (linux and cuda_compiler_version == "11.1")]
```
Comment (h-vetinari) [inline recipe/meta.yaml:49]: Anything below 11.2 has been droppped from conda-forge, so we can simplify this condition (and remove the comment above)

**Comment by h-vetinari** [comment]
> Resolves #41

Have you tested that this resolves #41? I tend to think that it doesn't even _need_ solving, see my comment there.

**Comment by mmcauliffe** [comment]
I have gotten the simple `conda build recipe` working from the repo root directory this way on windows.

**Comment by mmcauliffe** [comment]
I do also have https://github.com/conda-forge/kaldi-feedstock/pull/39 working now (was not expecting to get that working today), so it might be worth closing this and incorporating it and your comments into that one (specifically the cuda version ones)?

**Comment by h-vetinari** [comment]
> I have gotten the simple `conda build recipe` working from the repo root directory this way on windows.

OK, if that works, then I guess that doesn't hurt. I can bring over the changes to #39 if you want.

**Comment by mmcauliffe** [comment]
Changes incorporated in #39

**Pull request closed without merging**